go/common/crypto/signature/signer.go

```diff
@@ -17,6 +17,14 @@ const (
 )
 
 var (
+	// ComputeSignatureContext is the signature context used to sign compute
```

Comment:
I verify the `StorageReceipt` signature in `RuntimeGenesis.SanityCheck()` and I need `ComputeSignatureContext` for the context. To avoid circular dependencies I moved `ComputeSignatureContext` from `roothash/api/commitment` to `common/crypto/signature`.

Reply:
Oh hell no.  If you absolutely need to do something like this, carve out another sub-package under `roothash/api` for it.  Don't move context declarations into `common/crypto/signature`.

Reply:
I will not check the signatures becaues I also need public keys from the storage nodes which I don't have during the sanity check. I will move these contexts back and clean it up. For signature verification, there is a separate issue https://github.com/oasislabs/oasis-core/issues/1686

Reply:
Done.